### ISSUE TITLE
Training loss mismatch warning

### ISSUE THREAD

**Issue by dsamvoel** [hustvl/DiffusionDriveV2#29]
/home/user/data/Yongjae/navsim_workspace/navsim/navsim/agents/diffusiondrivev2/diffusiondrivev2_model_rl.py:1089: UserWarning: Using a target size (torch.Size([64, 1, 8, 3])) that is different to the input size (torch.Size([64, 80, 8, 3])). This will likely lead to incorrect results due to broadcasting. Please ensure they have the same size. trajectory_loss = F.l1_loss(diffusion_output, target_traj)

Is it normal case when I train the model_rl?

**Comment by GrootLiu** [comment]
I met the same warning
```
/data/DiffusionDriveV2/navsim/agents/diffusiondrivev2/diffusiondrivev2_model_rl.py:1018: UserWarning: Using a target size (torch.Size([64, 1, 8, 3])) that is different to the input size (torch.Size([64, 80, 8, 3])). This will likely lead to incorrect results due to broadcasting. Please ensure they have the same size.
  trajectory_loss = F.l1_loss(diffusion_output, target_traj)
```
